tests/ten/ten_cor_014/run.py

```diff
@@ -18,10 +18,26 @@ def execute(self):
         tx_receipt1 = network.transact(self, web3_usr, storage.contract.functions.store(1), account_usr, storage.GAS_LIMIT)
         tx_receipt2 = network.transact(self, web3_usr, storage.contract.functions.store(2), account_usr, storage.GAS_LIMIT)
 
+        # create another user and have them send funds to our test user
+        pk2 = self.get_ephemeral_pk()
+        web3_usr2, account_usr2 = network.connect(self, private_key=pk2, check_funds=False)
+        self.distribute_native(account_usr2, network.ETH_ALLOC_EPHEMERAL)  # Fund the second user
+    
+        # second user sends funds to our test user
+        transfer_amount = web3_usr2.to_wei(0.001, 'ether')
+        tx_receipt3 = network.transact(self, web3_usr2, {
+            'to': account_usr.address,
+            'value': transfer_amount,
+            'gas': 21000,
+            'gasPrice': web3_usr2.eth.gas_price,
+            'chainId': web3_usr2.eth.chain_id
```

Comment:
You most likely can't assume the gas will be 21000 on the L2 so should estimate it I think i.e

tx = {'to': account_usr.address,'value': transfer_amount,'gasPrice': web3_usr2.eth.gas_price}
tx['gas'] = web3.eth.estimate_gas(tx)

Reply:
much cleaner, hadn't noticed the checkFunds param either!